client/js/components/procedure/StatementSegmentsList/SegmentLocationMap.vue

```diff
@@ -241,6 +241,22 @@ export default {
       })
     },
 
+    /**
+     * Restore non-updatable comments from segments relationships after update request
+     */
+    restoreComments (comments) {
+      if (comments) {
+        const segmentWithComments = {
+          ...this.segment,
+          relationships: {
+            ...this.segment.relationships,
+            comments: comments
+          }
+        }
+        this.setItem({ ...segmentWithComments, id: this.segment.id })
```

Comment:
why do You need `this.segment.id` ? Shouldn't it be already set with `this.segment` ?

Reply:
Seems logical and it works without the additional id, so I removed it in [c88767f](https://github.com/demos-europe/demosplan-core/pull/3468/commits/c88767f0073d4e9650963acedb7281eefc944899)